tox.ini

```diff
@@ -92,8 +93,9 @@ setenv =
     init: SCENARIO="init_scenario_test"
     e2e: SCENARIO="e2e"
     rpsaas: SCENARIO="rpsaas"
+    secretsync: SCENARIO="secretsync_test"
     edge: SCENARIO="not rpsaas and not init_scenario_test"
```

Comment:
If `secretsync_test` requires prereqs on the cluster, I'm thinking we might want to modify the `edge` tox scenario to also exclude this test. While it is edge-side, it requires setup that's technically optional for all other edge tests. Thoughts?

Reply:
I was thinking the not rpsaas was going to handle that part already ...

but if we remove the rpsaas/consolidate the markers for said test, then I will need to change edge